single_level.py

```diff
@@ -27,14 +27,59 @@ def get_note_types(resource):
 		else:
 			note_types.append(note["jsonmodel_type"])
 	return note_types
+	
+def get_extent_number(resource):
+	extent_number = []
+	for extent in resource["extents"]:
+		if extent["number"] > 0:
+			extent_number.append("true")
+		else:
+			extent_number.append("false")
+	return extent_number
+	
```

Comment:
This might have been a result of trying to not get two "true" values added to the spreadsheet when there was multiple extents for a resource. I originally tried looking for an extent portion and I ran into problems (but that could have been a result of my inexperience writing code and not the field itself) - I didn't think about looking for a container summary because I thought it wasn't a required within "Extent" in ASpace (ASpace will let you save without having written a container summary).